features/HomeTab/FlatOffer/FlatOffer.tsx

```diff
@@ -0,0 +1,74 @@
+import { Octicons } from "@expo/vector-icons";
+// import { useEffect, useState } from "react";
+import { useContext } from "react";
+import { View } from "react-native";
+// import { DateData } from "react-native-calendars";
+import { ScrollView } from "react-native-gesture-handler";
+import { Text } from "react-native-paper";
+
+import Accordion from "./Accordion/Accordion";
+import BasicDetails from "./BasicDetails/BasicDetails";
+import Description from "./Description/Description";
+import Facilities from "./Facilities/Facilities";
+import Footer from "./Footer/Footer";
+import Gallery from "./Gallery/Gallery";
+import Header from "./Header/Header";
+import Map from "./Map/Map";
+import Owner from "./Owner/owner";
+import Reviews from "./Reviews/Reviews";
+import SettingsContext from "../../../contexts/SettingsContext";
+import translations from "../../../preferences/translations";
+
+export type CompanionType = "adults" | "children" | "pets";
+
+const FlatOffer = ({ route, navigation, bottomTabsRoute, bottomTabsNavigation, filters }) => {
+  const { flatOffer } = route.params;
+  const { settings } = useContext(SettingsContext);
+
```

Comment:
Bottom navigation tab should be hidden. Just add here (and import `useEffect`)
```
useEffect(() => {
  bottomTabsNavigation.setOptions({ tabBarStyle: { display: "none" } });

  // https://stackoverflow.com/a/64789273
  const unsubscribe = navigation.addListener("beforeRemove", (e: any) => {
    e.preventDefault();
    unsubscribe();
    bottomTabsNavigation.setOptions({ tabBarStyle: { display: "flex" } });
    navigation.navigate("FlatOfferList");
  });
}, [bottomTabsNavigation, navigation]);
```
![image](https://github.com/flatly-pw/mobile-app/assets/53649257/5752f30e-37d0-44bb-8c37-83c9065fb801)

![image](https://github.com/flatly-pw/mobile-app/assets/53649257/e60821d1-e554-48f8-a2e2-0cc17ea9d263)